.vale/styles/RedHat/CaseSensitiveTerms.yml

```diff
@@ -162,3 +160,114 @@ swap:
+  facter: Facter
+  filestore|File Store: FileStore
+  Firewalld|firewallD|FirewallD: firewalld
+  Greenboot|green boots: greenboot
+  GUI|HTTP interface: management console
+  hot rod|HotRod|hotrod: Hot Rod
+  Ignite|Fuse Ignite: Fuse Online|Red Hat Fuse Online|Syndesis
+  INSTALL_DIR|installDir: FUSE_HOME
+  Router: AMQ Interconnect
+  'Red\sHat\sInterconnect': AMQ Interconnect
+  'JBoss\sInterconnect': AMQ Interconnect
+  ipv4|IPV4|Ipv4: IPv4
+  ipv6|IPV6|Ipv6: IPv6
+  Data Grid console: Data Grid Console
+  JBoss Data Grid: Data Grid
+  kbase|knowledge base: KIE base
+  kernelspace: kernel space
+  key store: keystore
+  'Kie(?!\sServer)': KIE
+  'Kie\sServer': Intelligent Process Server
+  kie API|Kie API|knowledge API: KIE API
+  kjar|kJAR: KJAR
+  ksession|knowledge session: KIE session
+  Kubelet: kubelet
+  kubernetes|k8s: Kubernetes
+  Librados|LIBRADOS: librados
+  Librbd|LIBRBD: librbd
+  Lightweight Directory Access Protocol over Secure Socket Layer: LDAPS
+  live-backup group: master-slave group
+  master CA: IdM CA renewal server
+  Microsoft Azure Portal: Microsoft Azure portal
+  Microsoft Windows Server|Microsoft Windows|Windows(?! Server): Windows Server
+  network interface card: network interface controller
+  objectclass: objectClass
+  'OCM|(?<!Red Hat OpenShift )Cluster Manager|(?<!Red Hat )OpenShift Cluster Manager|the OpenShift Cluster Manager': Red Hat OpenShift Cluster Manager
+  OCP: Red Hat OpenShift Container Platform
+  ODF: Red Hat OpenShift Data Foundation
+  'OD|(?<!Red Hat OpenShift )Dedicated': Red Hat OpenShift Dedicated
+  '(?<!Microsoft Azure )On-Demand Marketplace': Microsoft Azure On-Demand Marketplace
+  Openshift online|OO: Red Hat OpenShift Online
+  '(?<!Red Hat )OpenStack Platform|RHOS|RH-OSP': Red Hat OpenStack Platform
+  Overcloud: overcloud
+  proof of concepts: proofs of concept (plural)
+  Properties editor: Properties View
+  puppet forge: Puppet Forge
+  puppet: Puppet
+  Qdmanage|QDMANAGE: qdmanage
+  Qdstat|QDSTAT: qdstat
+  Red Hat JBoss Data Grid|JDG: Red Hat Data Grid
+  Red Hat JBoss EAP: Red Hat JBoss Enterprise Application Platform
+  '(?<!Red Hat )JBoss Enterprise Application Platform': Red Hat JBoss Enterprise Application Platform
+  Data Grid Administration Console: Data Grid Console
+  '(?<!Business )Resource Planner|(?<!Business Resource )Planner': Business Resource Planner
+  RHDS: Red Hat Directory Server
+  Satellite server: Satellite Server
+  'self-hosted\sengine\svirtual\smachine|engine\sVM': Manager virtual machine
+  Smart State|smart state|Smart state|Smartstate|Analysis: SmartState analysis
+  Source Navigator: Source-Navigator^TM^
+  standard Manager|standard environment: standalone Manager
+  StartTLS|startTLS: STARTTLS
+  STI|source to image: Source-to-Image (S2I)
+  Subscription manifest: Subscription Manifest
+  Technology preview|technology preview: Technology Preview
+  The Operator Lifecycle Manager: Operator Lifecycle Manager
+  urn: URN
```

Comment:
Did the script miss generating entries for items like https://redhat-documentation.github.io/supplementary-style-guide/#vcpu

Reply:
hmmm. yes looks like it.

Reply:
Added the misssing terms. See below. There was a goof in the python logic.

Reply:
These updates picked up`vCPU` but it looks like an entry for`VDSM` is still missing.

Reply:
Oh actually VDSM is in Definitions.yml

Reply:
Python logic needs to be rewritten to handle these entries properly :/